packages/mrm/bin/mrm.js

```diff
@@ -163,29 +156,11 @@ Note that when a preset is specified no default search locations are used.`
 
 		if (isDefaultPreset) {
 			return [...paths, path.dirname(require.resolve('mrm-preset-default'))];
+		} else {
+			const presetPackageName = getPackageName('preset', preset);
+			const presetPath = await requirex.resolve(presetPackageName);
```

Comment:
Note to self: This may fail if preset is local, add a test for this case.

Reply:
Added the test, but it doesn't run the `resolveDirectories` function, which I wanted to test. Still more tests is good.